zephyr/include/sof/lib/regions_mm.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
can you explain what is the purpose of this commit and what is the heap allocator - inside the commit message for this patch?